lesson14/main.js

```diff
@@ -0,0 +1,83 @@
+"use strict";
+const wrapper = document.getElementById("wrapper");
+const lists = document.getElementById("lists");
+const createElBtn = document.getElementById("create-element-btn");
+createElBtn.classList.add("btn");
+const modal = document.getElementById("modal");
+const displayModal = document.getElementById("display-modal");
+const jsonURL = "https://myjson.dit.upm.es/api/bins/j16z";
+const removeModal = () => {
+  modal.remove();
+  wrapper.classList.remove("bg_gray");
+};
+
+const loadingIcon = document.createElement("img");
+const startLoading = () => {
+   loadingIcon.id = "loadingIcon";
+   loadingIcon.src = "img/loading-circle.gif";
+   wrapper.appendChild(loadingIcon);
+};
+
+const endLoading = () => loadingIcon.remove();
+
+displayModal.addEventListener("click", function () {
+   modal.style.display = "block";
+   wrapper.classList.add("bg_gray");
+   displayModal.remove();
+});
+
+createElBtn.addEventListener("click", () => {
+   const inputNumber = document.getElementById("number");
+   if (!inputNumber.value === false) {
+      console.log(inputNumber.value);
+      init();
+   } else {
+      alert("数字が未入力です。");
+   }
+});
```

Comment:
IMO
The if conditional expression, but it could be simply rewritten as ture.
```suggestion
modalBtn.addEventListener("click", () => {
  const inputNumber = document.getElementById("js-inputNumber");
  if (inputNumber.value) {
    console.log(inputNumber.value);
    init();
  } else {
    alert("数字が未入力です。");
  }
});
```

Reply:
Thank you for your advise. 
I changed the way conditional expression are written. But, it is not same that you suggested me.
So, please confirm that this way is also not bad.
a7c6d6c